### PR TITLE
chore(vscode): categories and keywords

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR adds few metadata to the `package.json` of the VScode extension.

Here's a link to the page that explains each field: https://code.visualstudio.com/api/references/extension-manifest

## Test Plan

None

### CODE REVIEW

File: editors/vscode/package.json [hunk 1]
```diff
@@ -35,6 +35,14 @@
...
 		}
 	},
+	"categories": [
+		"Formatters"
+	],
+	"keywords": [
+		"Formatter"
+	],
+	"license": "MIT",
+	"preview": true,
```
Comment (ematipico) [inline editors/vscode/package.json:45]: I decided to  mark it as a preview, as it's still a work a progress. What do you think?

Comment (leops) [reply editors/vscode/package.json:45]: I think it's okay for now, though eventually we'll probably want to patch the `package.json` on the fly to inject or remove this flag in CI before packaging depending on whether it's being built for a pre-release version or not

File: editors/vscode/package.json [hunk 2]
```diff
@@ -35,6 +35,14 @@
 			}
 		}
 	},
+	"categories": [
+		"Formatters"
+	],
+	"keywords": [
+		"Formatter"
```
Comment (MichaReiser) [inline editors/vscode/package.json:42]: Maybe add "JavaScript"?

Comment (ematipico) [reply editors/vscode/package.json:42]: We have a limit of 5 keywords, we should make them count 😛 We can also replace `Formatter` with `JavaScript`. What do you think?

**Review by MichaReiser — APPROVED** [review]

**Review by yassere — APPROVED** [review]
Looks good to me. I agree that we can refine our list of keywords later on.

**Pull request merged**